src/escapedHashtags.js

```diff
@@ -0,0 +1,189 @@
+            }
+        }
+        return parsed.toHashtag();
+    }
+
+    /*
+     * transforms escaped hashtags based on transformFn
+     * 
+     * transformFn -> function(input) where input will be text inside delimiters
+     * and returns what you want that turned into without delimiters
+     *
+     * example:
+     * transform("`#form/question`", makeBubble) -> "<bubble>#form</bubble>"
+     */
+    function transform(input, transformFn) {
+        if (!input) { return input; }
+        var symbols = getSymbols(input);
+        transformFn = transformFn || function (input) { return input; };
+        var state = OUTSIDE_HASHTAG,
+            strLen = symbols.length,
+            text = "",
+            currentReference = "";
+
+        for (var i = 0; i < strLen; i++) {
+            var current = symbols[i],
+                next = symbols[i+1];
+
+            if (state === OUTSIDE_HASHTAG) {
+                if (current === DELIMITER && next === DELIMITER) {
+                    text += DELIMITER;
+                    i++;
+                } else if (current === DELIMITER) {
+                    state = INSIDE_HASHTAG;
+                } else {
+                    text += current;
+                }
+            } else if (state === INSIDE_HASHTAG) {
+                if (current === DELIMITER && next === DELIMITER) {
+                    currentReference += DELIMITER;
+                    i++;
+                } else if (current === DELIMITER) {
+                    state = OUTSIDE_HASHTAG;
+                    text += transformFn(currentReference);
+                    currentReference = "";
+                } else if (next !== undefined){
+                    currentReference += current;
+                }
+            }
+        }
+
+        if (state === INSIDE_HASHTAG) {
+            // end of string, shouldn't happen, but will not
+            // overestimate users or Vellum devs
+            text += DELIMITER + currentReference;
+        }
+
+        return text;
+    }
+
+    // handle 4 byte unicode properly. pulled from
+    // https://mathiasbynens.be/notes/javascript-unicode#iterating-over-symbols
+    function getSymbols(string) {
+        var index = 0;
+        var length = string.length;
+        var output = [];
+        for (; index < length - 1; ++index) {
+            var charCode = string.charCodeAt(index);
+            if (charCode >= 0xD800 && charCode <= 0xDBFF) {
+                charCode = string.charCodeAt(index + 1);
+                if (charCode >= 0xDC00 && charCode <= 0xDFFF) {
+                    output.push(string.slice(index, index + 2));
+                    ++index;
+                    continue;
+                }
+            }
+            output.push(string.charAt(index));
+        }
+        output.push(string.charAt(index));
+        return output;
+    }
+
+    /*
+     * extends xpath parser to be aware of escaped hashtags
+     */
+    function Parser(hashtagDictionary) {
```

Comment:
Should not be capitalized. The convention is to capitalize constructor functions that are used to instantiate objects with the `new` operator.